internal/agent/install.go

```diff
@@ -218,8 +218,13 @@ func RunInstall(c *config.Config) error {
 
 	// UKI path. Check if we are on UKI AND if we are running off a cd, otherwise it makes no sense to run the install
 	// From the installed system
-	if internalutils.UkiBootMode() == internalutils.UkiRemovableMedia {
-		return runInstallUki(c)
+	if internalutils.IsUki() {
+		c.Logger.Debugf("UKI mode: %s\n", internalutils.UkiBootMode())
```

Comment:
bit more of logging to be able to check what mode are we in, in case we broke it like I already did :D